client/src/components/DataImportExport.vue

```diff
@@ -157,6 +157,13 @@ export default defineComponent({
       width="500"
     >
       <v-card>
+        <v-btn
+          icon
+          style="float:right"
```

Comment:
I think using float:right is deprecated at this point, but I'm not enough of a vuetify or CSS expert to say exactly what to do instead. @dchiquito can you comment on this?